### ISSUE TITLE
Normalizer is missing when querying by aria-label

### ISSUE THREAD

**Issue by johnjesse** [testing-library/dom-testing-library#854]
- `@testing-library/dom` version: latest

### Relevant code or config:

https://github.com/testing-library/dom-testing-library/blob/accb6cc60628cb6b5bd4d9be2ead41724995e5ae/src/queries/label-text.js#L94

The normalizer is missing as an option passed into the `queryByAllAttribute` function - and so it's ignored if you're looking for an element by the aria-label

I'm going to add a PR to fix this, but I wanted to raise the issue first for some visibility
